recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-core/dbus/dbus-cxx_0.10.0.bb

```diff
@@ -0,0 +1,16 @@
+DESCRIPTION = "C++ bindings for dbus"
```

Comment:
This should be split: meta-xt-images + bbappend

Reply:
Done